paper.md

```diff
@@ -0,0 +1,77 @@
+---
+title: 'Turftopic: Topic Modelling with Contextual Embeddings'
+tags:
+  - Python
+  - topic modelling
+  - sentence-transformers
+  - embeddings
+authors:
+  - name: Márton Kardos
+    orcid: 0000-0001-9652-4498
+    affiliation: 1
+  - name: Kenneth C. Enevoldsen
+    orcid: 0000-0001-8733-0966
+    affiliation: 1
+  - name: Jan Kostkan
+    orcid: 0000-0002-9707-7121
+    affiliation: 1
+  - name: Ross Deans Kristensen-McLachlan
+    orcid: 0000-0001-8714-1911
+    affiliation: "1, 3"
+  - name: Roberta Rocca
+    orcid: 0000-0002-0680-7097
+    affiliation: 2
+affiliations:
+ - name: Center for Humanities Computing, Aarhus University, Denmark
+   index: 1
+ - name: Interactive Minds Center, Aarhus University, Denmark
+   index: 2
+ - name: Department of Linguistics, Cognitive Science, and Semiotics, Aarhus University, Denmark
+   index: 3
+date: 17 March 2025
+bibliography: paper.bib
+---
+
+# Summary
+
+Turftopic is a topic modelling library including a number of recent topic models that go beyond bag-of-words and can understand text in context, utilizing representations from transformers.
+The library focuses on ease-of-use, providing a unified, interface for a number of different modern topic models, and boasting both model-specific and model-agnostic interpretation and visualization utilities.
+The user is afforded great flexibility in model choice and customization, but the library comes with reasonable defaults, not to overwhelm first-time users with a plethora of choices.
+In addition, our library is capable of modeling topics over time, modeling topics in streams of texts, finding hierarchical themes, and multilingual usage.
+For ease of interpretation, users can utilize large language models to assign human-readable names to topics, and can model key phrases, lemmas or stems right out of the box.
```

Comment:
The last part here is a bit unclear to the uninitiated reader - would reformulate